docs/user-guide/user-guide.rst

```diff
@@ -1260,6 +1260,70 @@ Click the Create Asset button to create an Asset manually. Enter relevant Asset
 
 -------------------------------
 
+Create Asset Batch
+~~~~~~~~~~~~~~~

```

Comment:
This section is not being built.  Tildes under the section title need to be longer than the header itself.